modules/scanning-images.adoc

```diff
@@ -20,6 +20,43 @@ Scanner identifies the vulnerabilities in the:
 * programming language specific dependencies
 * programming runtimes and frameworks
 
+[discrete]
+== Understanding and addressing the commonly occurring Scanner warning messages
+
+When scanning images with {rh-rhacs-first}, you might see the `CVE DATA MAY BE INACCURATE` warning message.
+Scanner displays this message when it cannot retrieve complete information about the operating system or other packages in the image.
+
+This occurs because the image is based on an operating system or package manager that Scanner does not support, or because the operating system or package manager version has reached end-of-life and no longer receives security updates.
+
+It is important to address this warning to ensure that vulnerability reports are as accurate as possible.
```

Comment:
`vulnerability reports` could be mistruted to the ACS vulnerability reporting feature. I suggest changing the language. For example, `vulnerability analysis is as accurate as possible` .